src/Sarif.Viewer.VisualStudio/Sarif/Invocation.extensions.cs

```diff
@@ -25,7 +25,7 @@ public static InvocationModel ToInvocationModel(this Invocation invocation)
                 Account = invocation.Account,
                 ProcessId = invocation.ProcessId,
                 FileName = invocation.ExecutableLocation?.Uri?.ToString(),
-                WorkingDirectory = invocation.WorkingDirectory,
+                WorkingDirectory = invocation.WorkingDirectory?.Uri?.ToString(),
```

Comment:
>ToString [](start = 69, length = 8)

Perhaps `.OriginalString`?

Reply:
I was following what FileName does. Not really important, as this property isn't currently used.

---
In reply to: [217543302](https://github.com/Microsoft/sarif-visualstudio-extension/pull/34#discussion_r217543302) [](ancestors = 217543302)